checks.c

```diff
@@ -418,3 +427,197 @@ void hibachk_authorized_users(const struct hibaenv *env, const struct hibacert *
 	fflush(f);
 	sshbuf_free(options);
 }
+
+char base46_map[] = {'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P',
+                     'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 'a', 'b', 'c', 'd', 'e', 'f',
+                     'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v',
+                     'w', 'x', 'y', 'z', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '+', '/'};
+
+char* base64_decode(char* cipher) {
```

Comment:
There is already a base64 decode implementation in sshlib, that HIBA links against. No need to reimplement.